Volume/Views/MainView/PublicationList.swift

```diff
@@ -79,7 +79,7 @@ struct PublicationList: View {
                     case .results(let results):
                         LazyHStack(spacing: 12) {
                             ForEach(results.followedPublications) { publication in
-                                NavigationLink(destination: PublicationDetail(publication: publication)) {
+                                NavigationLink(destination: PublicationDetail(entryPoint: .followingPublications, publication: publication)) {
```

Comment:
You put entry point after the article/publication in other places. Keep it consistent. 

For example, see BookmarksList line 69. I like it with entry point after the model (with the idea of putting the most important parameter first). That consistency outweighs alphabetizing or adding an explicit initializer in my opinion.